lib/main.js

```diff
@@ -12,6 +12,8 @@ import ReactGA from 'react-ga'
 
 // import OTP-RR components
 import { createOtpReducer } from 'otp-react-redux'
+import { createHashHistory } from 'history'
+import { connectRouter, routerMiddleware } from 'connected-react-router'
```

Comment:
whoops, looks like we actually do need connected-react-router in the package.json for this library.

Reply:
history too

Reply:
Fixed.